metrics/processors/node_autoscaling_enricher.go

```diff
@@ -73,9 +72,7 @@ func (this *NodeAutoscalingEnricher) Process(batch *core.DataBatch) (*core.DataB
 			setFloat(metricSet, &core.MetricNodeMemoryCapacity, float64(capacityMem.Value()))
 			setFloat(metricSet, &core.MetricNodeMemoryAllocatable, float64(allocatableMem.Value()))
 
-			if storageExist && allocatableStorageExist {
-				setFloat(metricSet, &core.MetricNodeEphemeralStorageCapacity, float64(capacityEphemeralStorage.Value()))
```

Comment:
Here you are removing this and later you're trying to process this metric. Why?

Reply:
I've moved this metrics from based on API Server to based on Summary API. Can you clarify what's the concern?

Reply:
Capacity and allocatable values are not available in the Summary API as far as I know. Did you verify they are there for ephemeral storage?

Reply:
Yes, they are available and the same as capactiy and allocatable that API Server uses.